src/apps/src/main/scala/osmesa/apps/batch/ChangesetStats.scala

```diff
@@ -1,4 +1,4 @@
-package osmesa.analytics.oneoffs
+package osmesa.apps.batch
```

Comment:
Is this a good candidate to create an `osmesa.apps.example` package for?  This utility doesn't fit into the workflow as such.  It only creates CSV output, though you could suggest that the CSVs could be used to ingest a  DB as well as the direct write operation.  One way or another, it's not part of our process anymore, so we either move this out to a less confusing place, or we rename to something less confusing, like `IngestToCSV`.

Reply:
Going to reach out to you on slack to address this one, there's a few potentially interconnected topics at play here that relate to what you've brought up.

Reply:
Summary of followup:

This file is out of date, generating a db schema we no longer use and is therefore of extremely limited utility. We elected to just delete this file instead and remove its deployment ref from the batch Makefile.